### DISCUSSION TITLE
Change the time of the validation ceremony

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-docs#61] [IIPs]
Do you agree to change the time of the validation ceremony from 13:30 UTC to 15:00 UTC?

Motivation: this proposal is aimed to make the time of the validation ceremony more convenient for people located in the US. There's a lot of active crypto users in the western part of the US. Now they have validation at 5:30 AM which is very inconvenient. That is a major limiting factor for the adoption of Idena in the US.

New validation time for different time zones: 

- GMT 15:00
- San Francisco, USA 8:00
- New York, USA 11:00
- Tunis, Tunisia 16:00
- Berlin, Germany 17:00
- Moscow, Russia 18:00
- Delhi, India 20:30
- Jakarta, Indonesia 22:00
- Beijing, China 23:00
- Sydney, Australia 01:00 + 1 day
- Auckland, New Zealand 03:00 + 1 day
- Honolulu, Hawaii, USA 05:00 + 1 day

**Comment by Syd-ai** [comment]
I agree with this proposal, the pros seem to overtake the cons

**Reply by rioda-org** [reply]
Makes sense to me. I think that we would gain larger population with this move.

**Comment by ubiubi18** [comment]
UTC 15:00 seems indeed more convenient. 

(Please check all times according to current time changes, i think it is san francisco 7 am, not 8 am? Also Berlin 4 pm, not 5 pm?)

**Comment by hngts** [comment]
Yes, for my timezone, both 13:30 and 15:00 UTC - perfectly doable.

**Comment by idena-Egyptian-Pharoh** [comment]
It will be in egypt 17:00 seems good to egyptian people even in ramadan fast will be good time 1.5 hour before eating

**Comment by YogaSakti** [comment]
I disagree, the current validation clock is right in my opinion. if it is changed it will be too late for some Asian countries and Australians, 5:30 AM is good enough for the western part of US what do you mean by "very inconvenient"?

You should have a better reason than "very inconvenient". Do you think 11:30 PM or 01:00 AM for australian is very convenient?

**Comment by TemplarMiner** [comment]
Changing of validation timing should be proposed and implemented for the one after tomorrow. Too rush the decision and too abruptly executed. Not many will take note of this changes.

**Reply by rioda-org** [reply]
I think something like this needs to be done using new version of node and hard fork voting. Even if Oracle voting shows that people want to vote for change, they will be able to stop it by not updating their nodes to latest version.

**Reply by sekisanchi** [reply]
Completely agreed.  Please don't change. Those type of fundamental change have to be done on separate chain. For Asian countries, the shift will reduce almost half of prospective volume in my estimation, while not to affect much on US or North/South America's.  Also it's catastrophic for Japan, where I'm focusing.

**Comment by 0xpatel57** [comment]
make sense to me, so that I can wake up later on Saturday morning!!

**Comment by hlolve** [comment]
Fixed time is good to remember but always will be good for ones and bad for others. Maybe validation time needs rotate to be more fair, one validation 20:00 UTC next epoch 8:00 UTC and then back to 20:00 UTC.

**Reply by singulart** [reply]
See my reply, I also believe that fixed time is bad

**Comment by sekisanchi** [comment]
I have serious concerns upon the voting itself- 

1) No rational statistics provided upon the number of prospective people. Population per time zone and activity curve on them have to be shown at least, to evaluate the impact of the clock shift. 
2) The questionnaire itself is heavily biased to one direction. If you asked that the current clock is allowable, most will answer yes. 

Thus, I strongly recommend to invalidate current vote, find and provide rational statistics to consider the clock change, and voting it again with a improved fair questionnaire.

**Comment by singulart** [comment]
No matter what hour is chosen, since it's a global setting, it cannot satisfy everyone. Someone from the validators community will  be sleeping at that time anyway! 

I cannot stress this strong enough: ability to choose the validation time **individually** is essential for idena. Please consider adding this feature.

**Reply by hlolve** [reply]
"validation time individually" is not possible, will let a person to make multiple accounts
